keps/sig-storage/3762-persistent-volume-last-phase-transition-time/README.md

```diff
@@ -676,13 +651,20 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+Rollout should not fail and there should be no need for rollback as this enhancement only adds a new field.
```

Comment:
Well - not very likely, but one can imagine a some nil-pointer exception or sth like that (i.e. crashes of kube-apiserver when it tries to set it).

Reply:
Sure, rewording a bit.